src/entities/Proposal/templates/messages.ts

```diff
@@ -14,8 +14,8 @@ export function getUpdateMessage(proposal: ProposalAttributes, votes: Record<str
       throw Error("Proposal can't be enacted without an enacting user")
     }
     let description: string | null = ''
-    if (proposal.vesting_address) {
-      description += 'Vesting Contract Address: ' + proposal.vesting_address
+    if (proposal.vesting_addresses.length > 0) {
+      description += 'Vesting Contract Addresses: ' + proposal.vesting_addresses
```

Comment:
Shouldn't we post here the last/active one?

Reply:
In this instance it is not possible to know if, for example, a new vesting was added in case there was already one. So I would always show the total